public/styles/Header.scss

```diff
@@ -0,0 +1,140 @@
+@import '~bootstrap/scss/functions';
+@import '~bootstrap/scss/variables';
+@import '~bootstrap/scss/mixins';
+
+header{
+  background-color: $white-color;
+  color: $nav-item-black;
+
+  .navbar {
+    @include outer-box-shadow(0, 0, 1px, 0, $nav-item-black);
+    text-transform: uppercase;
+
+    @include media-breakpoint-up(md){
+      font-size: 0.9em;
+      padding: 8px 100px;
+
+    }
+
+    @include media-breakpoint-down(md) {
+      .navbar-collapse {
+        color: $primary-color;
+        position: fixed;
+        top: 72px; /* Height of navbar */
```

Comment:
Same thing here. `px` is an absolute unit of measurement while `em` is a relative unit of measurement. Relative unit fit well across screens.

Reply:
This is fixed.